<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Poker Tournament Manager - by Kost</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/data/icon.svg</normaloff>:/data/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget
{
    /*background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #BBB);*/
	background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:0.00303828, y2:0.005, stop:0.0789474 rgba(0, 134, 25, 255), stop:0.703349 rgba(0, 98, 10, 255));
}

QGroupBox
{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex;
	font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6EDF6E, stop: 1 #0E6F0E);
    color: #fff;
}

QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 10px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #8a8b8e, stop: 1 #dadbde);
    color: #888;
}

QPushButton:enabled {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
    color: #000;
}

#startButton {
   border-radius: 35px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
      <item>
       <widget class="QLabel" name="labelChip_4">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="statusTip">
         <string>:-)</string>
        </property>
        <property name="pixmap">
         <pixmap resource="data.qrc">:/data/chip-black.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gboxTourney">
        <property name="title">
         <string>Dati Torneo</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tourneyName">
           <property name="statusTip">
            <string>Inserisci il nome del torneo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_1">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Giocatori:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>spinPlayers</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinPlayers">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="statusTip">
            <string>Numero di seats</string>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>10000</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelChip">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="statusTip">
         <string>:-)</string>
        </property>
        <property name="pixmap">
         <pixmap resource="data.qrc">:/data/chip-red.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gboxLevels">
      <property name="title">
       <string>Gestione Livelli</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableLevels">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Completa la tabella del torneo</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tempo</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ante</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Piccolo Buio</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Grande Buio</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rebuy</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,1">
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="levelUp">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="statusTip">
            <string>Rimuovi Livello selezionato</string>
           </property>
           <property name="text">
            <string>^</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="levelDown">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="statusTip">
            <string>Rimuovi Livello selezionato</string>
           </property>
           <property name="text">
            <string>v</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="levelAdd">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="statusTip">
            <string>Aggiungi livello</string>
           </property>
           <property name="text">
            <string>Aggiungi</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="levelRem">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="statusTip">
            <string>Rimuovi Livello selezionato</string>
           </property>
           <property name="text">
            <string>Rimuovi</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="levelEdit">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="statusTip">
            <string>Rimuovi Livello selezionato</string>
           </property>
           <property name="text">
            <string>Modifica</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gboxChips">
      <property name="title">
       <string>Gestione Chips</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1,0,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stack iniziale:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinChips</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinChips">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Livello iniziale dello stack</string>
         </property>
         <property name="maximum">
          <number>100000000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stack rebuy:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>spinChips</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinRebuyChips">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="statusTip">
          <string>Chip per il rebuy</string>
         </property>
         <property name="maximum">
          <number>100000000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,1,0">
      <item>
       <widget class="QLabel" name="labelChip_2">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>70</height>
         </size>
        </property>
        <property name="statusTip">
         <string>:-)</string>
        </property>
        <property name="pixmap">
         <pixmap resource="data.qrc">:/data/chip-white.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="statusTip">
         <string>Avvio del torneo</string>
        </property>
        <property name="text">
         <string>S T A R T</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelChip_3">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>70</height>
         </size>
        </property>
        <property name="statusTip">
         <string>:-)</string>
        </property>
        <property name="pixmap">
         <pixmap resource="data.qrc">:/data/chip-blue.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Aiuto</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Lingua</string>
     </property>
     <addaction name="actionLangIta"/>
     <addaction name="actionLangEng"/>
    </widget>
    <addaction name="menuLanguage"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;Nuovo</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Apri</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Salva</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Salva &amp;Come...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Esci</string>
   </property>
  </action>
  <action name="actionKost_Rulez">
   <property name="text">
    <string>Kost Rulez ...</string>
   </property>
  </action>
  <action name="actionLangIta">
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/data/flag_IT.jpg</normaloff>:/data/flag_IT.jpg</iconset>
   </property>
   <property name="text">
    <string>Italiano</string>
   </property>
  </action>
  <action name="actionLangEng">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="data.qrc">
     <normaloff>:/data/flag_EN.jpg</normaloff>:/data/flag_EN.jpg</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;Informazioni</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
